src/lib/bootloader/sections.rb

```diff
@@ -0,0 +1,39 @@
+require "yast"
+require "yast2/execute"
+
+module Bootloader
+  # Represents available sections and handling of default boot entry
+  class Sections
+    # @param [CFA::Grub2::GrubCfg] grub_cfg - loaded parsed grub cfg tree
+    def initialize(grub_cfg)
+      @data = grub_cfg.sections
```

Comment:
I think it is good style to declare `@default = nil` here. `ruby -w` will warn otherwise.